packages/nuxt/src/module.ts

```diff
@@ -12,15 +15,37 @@ export interface ModuleOptions {
    * @see https://onyx.schwarz/development/#installation
    */
   disableGlobalStyles?: boolean;
+  /**
+   * This module will automatically detect if @nuxtjs/i18n is used within the project and configure itself to work with it.
+   * This group of options gives you the flexibility to fine tune this integration.
+   */
+  i18n?: {
+    /**
+     * Prefix to use for all translation keys comming from onyx. The modules translations can later be used under this prefix, e.g. `$t('onyx.optional')`.
+     * @default "onyx"
+     */
+    prefix?: string;
+  };
 }
 
 export default defineNuxtModule<ModuleOptions>({
   meta: {
-    name: "onyx-nuxt",
+    name: "@sit-onyx/nuxt",
     configKey: "onyx",
   },
-  defaults: {},
+  defaults: {
+    i18n: { prefix: "onyx" },
+  },
   setup(options, nuxt) {
+    const logger = useLogger("@sit-onyx/nuxt");
+    const { resolve } = createResolver(import.meta.url);
+
+    nuxt.options.runtimeConfig.public = defu(nuxt.options.runtimeConfig.public, {
```

Comment:
This merges the defaults passed to the module inside the nuxt config or passed as env vars. Thereby making the final values available inside the application and also the runtime parts of this module like the nuxt plugin.

Reply:
But why would we even need to modify the runtime config? I guess only env variables should be added there so I don't see a reason currently why this is needed.

If this is only needed to support changing the "onyx" prefix, I would prefer to remove the option instead like already suggested since I don't like manipulating the users runtime config :)

Reply:
Ok, fine then the i18n scope will simply no longer be configureable. But so you know, the runtime config is not just for env vars. It's a common practice for modules to store configs in there giving users the option to overwrite them using env vars and more importantly providing them to the runtime part of the module. Otherwise e.g. plugins would not be able to access any of the options passed to the module.